### PR TITLE
Correct some strange word usage

### PR DESCRIPTION
**Pull request by Woebin**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@Woebin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ide/reference/how-to-use-the-keyboard-exclusively.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ For a full listing of command shortcut keys in Visual Studio, see [Default keybo
 ::: moniker-end
 
 > [!NOTE]
-> Depending on your settings or the edition of Visual Studio you use, the dialog boxes and menu commands you see might differ from those described in Help. To change your settings, choose **Import and Export Settings** on the **Tools** menu. For more information, see [Reset settings](../environment-settings.md#reset-settings).
+> Depending on your settings or the edition of Visual Studio you use, the dialog boxes and menu commands you see might differ from those described in Help. To change your settings, select **Import and Export Settings** in the **Tools** menu. For more information, see [Reset settings](../environment-settings.md#reset-settings).
```
Comment (TerryGLee) [inline docs/ide/reference/how-to-use-the-keyboard-exclusively.md:34]: We appreciate your taking the time to create a PR! We'd like to ask for a few changes, though. Starting with the following:

Please note that our convention is that items are "on" a menu. See the examples in here: [https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#multiple-input-methods-and-branching-within-procedures](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#multiple-input-methods-and-branching-within-procedures)

File: docs/ide/reference/how-to-use-the-keyboard-exclusively.md [hunk 2]
```diff
@@ -65,17 +65,17 @@ To move among the options in a dialog box and change option settings by using on
 
 ## Navigate between windows and files
 
-- To move among files in an editor or designer, choose the **Ctrl**+**Tab** keyboard shortcut to display the IDE Navigator with **Active Files** selected. Choose the **Enter** key to navigate to the highlighted file.
+- To move among files in an editor or designer, use the **Ctrl**+**Tab** keyboard shortcut to display the IDE Navigator with **Active Files** selected. Use the **Enter** key to navigate to the highlighted file.
```
Comment (TerryGLee) [inline docs/ide/reference/how-to-use-the-keyboard-exclusively.md:68]: Please change instances of "Use the **Enter** key" to "Press the **Enter** key." In other words, we use the verb "press" in an imperative statement regarding specific keys on a keyboard. (The first instance of "use the _x_ shortcut" is fine, however.) 

This is also per examples in the guidance link listed in the previous comment.

File: docs/ide/reference/how-to-use-the-keyboard-exclusively.md [hunk 3]
```diff
@@ -65,17 +65,17 @@ To move among the options in a dialog box and change option settings by using on
...
 
-- To move among docked tool windows, choose the **Alt**+**F7** keyboard shortcut to display the IDE Navigator with **Active Tool Windows** selected. Choose the **Enter** key to navigate to the highlighted window.
+- To move among docked tool windows, use the **Alt**+**F7** keyboard shortcut to display the IDE Navigator with **Active Tool Windows** selected. Use the **Enter** key to navigate to the highlighted window.
```
Comment (TerryGLee) [inline docs/ide/reference/how-to-use-the-keyboard-exclusively.md:70]: Same comment about "Use the Enter key" ... please change the verb "use" to "press" . Thanks!

**Review by TerryGLee — CHANGES_REQUESTED** [review]

**Comment by TerryGLee** [comment]
@Woebin Thanks again for your PR. We requested changes to your original PR but haven't heard back; so, to address that which you've noted, we've created our own internal PR to capture most of your suggested changes, which will be published live after 4:00pm PDT today (10/16/2020). Meanwhile, we'll close this PR, with our thanks to you for helping to improve our documentation.

**Comment by Woebin** [comment]
> @Woebin Thanks again for your PR. We requested changes to your original PR but haven't heard back; so, to address that which you've noted, we've created our own internal PR to capture most of your suggested changes, which will be published live after 4:00pm PDT today (10/16/2020). Meanwhile, we'll close this PR, with our thanks to you for helping to improve our documentation.
Hi! I'm really sorry, it seems I either didn't get notification emails for the comments or they ended up in spam, so I honestly just forgot about this until right now. I'm absolutely in favor of you taking my suggestions and using them in a separate PR though, I'm just glad I could be of help even though I failed to respond in a timely manner (again, sorry about that; I'll have to check my notification settings for the future).

**Comment by TerryGLee** [comment]
@Woebin No worries! We always appreciate PRs that folks send our way. We look forward to hearing from you again. :)

**Pull request closed without merging**